models/engine/db_storage.py

```diff
@@ -74,3 +74,17 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Method to retrieve one object"""
+        if classes.get(cls) is None:
+            return (None)
+
+        for obj in self.__session.query(classes.get(cls)).filter(
+                classes.get(cls).id == id):
+            return (obj)
+        return (None)
+
+    def count(self, cls=None):
+        """Method to count the number of objects in storage"""
+        return (len(self.all(cls)))
```

Comment:
Because of the SQL `COUNT` function (supported by SQLAclehmy), you don't have too load all the objects into memory to count them as you did here.